### ISSUE TITLE
Client doesn't open on macOS Sierra

### ISSUE THREAD

**Issue by jackiehluo** [pritunl/pritunl#406]
Just upgraded my OS to macOS Sierra, and Pritunl no longer opens. The program starts and appears to be running, but I can't open the window to connect to the VPN.

Other Electron apps I have are unaffected by the upgrade.

**Comment by zachhuff386** [comment]
The process `pritunl-service` should always be running if not run `sudo pritunl-service` to determine why it isn't running.

Check the log file at `~/Library/Application Support/pritunl/pritunl.log`

Run the command `/Applications/Pritunl.app/Contents/MacOS/Electron` from the terminal.

**Comment by jackiehluo** [comment]
@zachhuff386 Thanks for the quick response!

`pritunl-service` is returning this error:
```
[ERRO] ▶ main: Server error
listen tcp 127.0.0.1:9770: bind: address already in use
panic: listen tcp 127.0.0.1:9770: bind: address already in use
```

That log file doesn't exist for me. And the Electron command just returns `[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0`.

**Comment by zachhuff386** [comment]
The issue is with macOS Sierra, possibly an open file limit issue. The following code will stop working after a few seconds. This causes the `pritunl-service` to stop accepting requests and prevents the client dialog from opening. Going to wait until the stable Sierra release to do additional testing.

```go

package main

import (
	"github.com/gin-gonic/gin"
)

func main() {
	engine := gin.Default()
	engine.GET("/ping", func(c *gin.Context) {
		c.String(200, "")
	})
	engine.Run("127.0.0.1:9770")
}
```

```bash
#!/bin/bash
while true
do
	curl http://localhost:9770/ping
	sleep 1
done
```

**Comment by zachhuff386** [comment]
This is fixed in v0.10.19
